CONTRIBUTING.md

```diff
@@ -34,6 +34,30 @@ If you consider opening a PR, here is some documentation to get you started:
 - [deck.gl API design guidelines](/dev-docs/deckgl-api-guidelines.md)
 
 
+## Working with other vis.gl dependencies
+
+Deck.gl has a number of dependencies that fall under vis.gl, and there may be times when it is necessary to make a change in one of these.
+Thus for development it is necessary to checkout a copy of such a dependency and make local changes.
+
+To get the local build of deck.gl to pick up the local code rather than the modules from npm, there are two important config files:
+
+- `ocular-dev-tools.config.js`
+- `examples/webpack.config.local.js`
+
+In `ocular-dev-tools.config.js` there is `aliases` config option which can be used to pull in specific dependencies.
```

Comment:
I would like to keep this document concise. There are many things going on in the ocular dev tools config and we can't explain them all here. For those who are interested in this part of the set up, we can simply link to ocular-dev-tools [documentation](https://uber-web.github.io/docs/dev-tools), and move the explanation and example to comments inside `ocular-dev-tools.config.js`.

Reply:
Agreed. No good if one can't see the wood for the trees. I've stripped the doc down, but did leave an example of an alias as I find myself constantly wanting to copy-paste this